pages/reference/base-images/base-images.md

```diff
@@ -18,15 +18,15 @@ excerpt: Docker images maintained by {{ $names.company.lower }}
 	- i386
 - Multiple Distributions:
 	- [Debian](https://www.debian.org/): jessie, sid, stretch and buster
-	- [Alpine](https://alpinelinux.org/): 3.5, 3.6, 3.7, 3.8, 3.9, 3.10 and edge
-	- [Ubuntu](https://www.ubuntu.com/): trusty (14.04), xenial (16.04), artful (17.10), bionic (18.04), and cosmic (18.10)
-	- [Fedora](https://getfedora.org/): 26, 28, 29 and 30
+	- [Alpine](https://alpinelinux.org/): 3.6, 3.7, 3.8, 3.9, 3.10 and edge
+	- [Ubuntu](https://www.ubuntu.com/): xenial (16.04), bionic (18.04), cosmic (18.10), disco (19.04) and eoan (19.10)
+	- [Fedora](https://getfedora.org/): 26, 28, 29, 30 and 31
 - Multiple language stacks:
-	- [Node.js](https://nodejs.org/en/): 12.7.0, 11.15.0, 10.16.1, 8.16.1 and 6.17.1
-	- [Python](https://www.python.org/): 2.7.16, 3.5.7, 3.6.9 and 3.7.4
-	- [openJDK](https://openjdk.java.net/): 7-jdk/jre, 8-jdk/jre, 10-jdk/jre and 11-jdk/jre
-	- [Golang](https://golang.org/): 1.12.7, 1.11.12 and 1.10.8
-	- [Dotnet](https://docs.microsoft.com/en-gb/dotnet/core/): 2.2-sdk/runtime/aspnet,3.0-sdk/runtime/aspnet,2.1-sdk/runtime/aspnet
+	- [Node.js](https://nodejs.org/en/): 13.1.0, 12.13.0, 11.15.0, 10.17.0, 9.11.2, 8.16.1 and 8.11.1
+	- [Python](https://www.python.org/): 2.7.17, 3.8.0, 3.7.5, 3.6.9, 3.5.7 and 3.4.10
```

Comment:
2.7.x, 3.8.x, 3.7.x, 3.6.x, 3.5.x and 3.4.x looks like a strange version-ordering system? :wink: 

Also, it seems some version-lists here are sorted largest -> smallest (e.g. Node.js), and others are sorted smallest -> largest (e.g. openJDK) ?